src/vs/workbench/contrib/tasks/electron-browser/task.contribution.ts

```diff
@@ -569,15 +569,15 @@ class TaskService extends Disposable implements ITaskService {
 		});
 
 		CommandsRegistry.registerCommand('workbench.action.tasks.reRunTask', (accessor, arg) => {
-			this.reRunTaskCommand(arg);
+			this.reRunTaskCommand(Types.isArray(arg) ? arg[0] : arg);
 		});
 
 		CommandsRegistry.registerCommand('workbench.action.tasks.restartTask', (accessor, arg) => {
-			this.runRestartTaskCommand(arg);
+			this.runRestartTaskCommand(Types.isArray(arg) ? arg[0] : arg);
 		});
 
 		CommandsRegistry.registerCommand('workbench.action.tasks.terminate', (accessor, arg) => {
-			this.runTerminateCommand(arg);
+			this.runTerminateCommand(Types.isArray(arg) ? arg[0] : arg);
```

Comment:
not needed

Reply:
Terminate only pays attention to one string. If an array is passed in, then it can only handle the first element.

Reply:
Fixed by allowing string as the args.